src/work-allocation/containers/available-tasks/available-tasks.component.ts

```diff
@@ -100,22 +123,18 @@ export class AvailableTasksComponent extends TaskListWrapperComponent {
 
   /**
    * Handle a User Claiming a Task
-   *
-   * TODO: This function will need to handle 'Assign to me and go to case' - EUI-2963.
    */
   public onActionHandler(taskAction: InvokedTaskAction): void {
-
     switch (taskAction.action.id) {
       case TaskActionIds.CLAIM:
-        return this.claimTask(taskAction.task.id);
```

Comment:
This is cleaner as a return. It then doesn't need the `break;`. Same for the two below.

Reply:
Agree, think this is a merge issue